app/assets/javascripts/les/profile_select_box.js

```diff
@@ -36,7 +36,7 @@ var ProfileSelectBox = (function () {
             key = defaultValue.replace(/default/, '').toLowerCase();
 
             if (key === 'capacity') {
-                key = 'electrical_capacity';
+                key = 'carrier_capacity';
```

Comment:
@antw would these three lines still be needed now that's decided that "Output capacity" (`capacity`) is displayed in the technology matrix instead of `electrical_capacity`?